docs/spark/how-to-guides/dotnet-spark-jupyter-notebooks.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Use Jupyter notebooks
+titleSuffix: .NET for Apache Spark
+description: Use .NET for Apache Spark in interactive environments like Jupyter Notebook, Jupyter Lab, or Visual Studio Code (VS Code)
+ms.author: luquinta
+author: luisquintanilla
+ms.date: 10/09/2020
+ms.topic: conceptual
+ms.custom: mvc, how-to
+---
+
+# Use .NET for Apache Spark in Jupyter notebooks
+
+In this article, you learn how to run .NET for Apache Spark jobs interactively in Jupyter Notebook and Visual Studio Code (VS Code) with .NET Interactive.
+
+## About Jupyter
+
+[Jupyter](https://jupyter.org/) is an open-source, cross-platform computing environment that provides a way for users to prototype and develop applications interactively. You can interact with Jupyter through a wide variety of interfaces such as Jupyter Notebook, Jupyter Lab, and VS Code.
+
+In the context of .NET, [.NET Interactive](https://github.com/dotnet/interactive), a .NET Core global tool, provides a kernel for writing .NET code (C#/F#) in interactive computing environments such as Jupyter Notebook.
+
+## Prerequisites
+
+- [.NET Core 3.1 SDK](https://docs.microsoft.com/dotnet/core/install/)
+- Apache Spark
+- Apache Spark .NET Worker
+
+See the [getting started tutorial](../tutorials/get-started.md) for more information on setting up your .NET for Apache Spark environment.
+
+## Prepare environment
+
+To work with Jupyter Notebooks, you'll need two things.
+
+1. Install [.NET Interactive global .NET tool](https://github.com/dotnet/interactive/blob/main/docs/NotebooksLocalExperience.md)
+1. Download `Microsoft.Spark` NuGet package.
+    1. Navigate to the [Microsoft.Spark](https://www.nuget.org/packages/Microsoft.Spark/) NuGet package page.
+
+        > [!IMPORTANT]
+        > By default, the latest version of the package is downloaded. **Make sure that the version you download is the same as your Apache Spark .NET Worker.**
+
+    1. In the **Info** pane, select **Download package**. Doing so downloads the latest version of the package to a file with a name similar to  *microsoft.spark.[PACKAGE-VERSION].nupkg*.
+    1. Unzip the recently downloaded package. The unzipped directory should contain a subdirectory called *jars*. Take note of the path since it's used at a later time.
+
+## Start .NET for Apache Spark
+
+Start .NET for Apache Spark in debug mode. This will will set up a process that will wait for connections from a [SparkSession](xref:Microsoft.Spark.Sql.SparkSession). Make sure to provide the path to the jar for the respective version of Apache Spark you're using.
+
```

Comment:
```suggestion
Start .NET for Apache Spark in debug mode. This will will set up a process that will wait for connections from a [SparkSession](xref:Microsoft.Spark.Sql.SparkSession). Make sure to provide the path to the `microsoft-spark-<version>.jar` for the respective version of .NET for Apache Spark you're using.

```

Reply:
done